Formula/config_provider.rb

```diff
@@ -0,0 +1,33 @@
+require 'json'
+
+class ConfigProvider
+    ##
+    # Creates a ConfigProvider which helps in reading values from bottle-config.json
+    # We are moving values to json file so that it is easy to modify them by bot
+
+    CONFIG_FILE = File.join(File.dirname(__FILE__), "..", "bottle-config.json")
+    def initialize(config_file_path=CONFIG_FILE)
+        file = File.read(config_file_path)
+        @config_data = JSON.parse(file)
```

Comment:
I think so long as we are fully automating this, can we have some sort of in-code check that the values make sense against each other?

For example, seems like we could have a hook here to download the root URL, and check the SHA.

Reply:
Actually I was planning to do this when automatically updating the config file through our bot.

Reply:
This makes more sense, I wasn't thinking about when this is run.